### PR TITLE
Vberenz/status

### PR DESCRIPTION
**Pull request by vincentberenz**
#what
moved status to its own hpp file, directly in the robot interfaces namespace

#checked if ok
only checked success of compilation

### CODE REVIEW

File: include/robot_interfaces/status.hpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#pragma once
```
Comment (luator) [inline include/robot_interfaces/status.hpp]: The file should probably have a license header like the other ones.

File: include/robot_interfaces/status.hpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+#include <string>
+#include <vector>
+#include <robot_interfaces/loggable.hpp>
+
+namespace robot_interfaces
+{
+
+  struct Status : public Loggable
```
Comment (luator) [inline include/robot_interfaces/status.hpp]: Indentation is not following the style guide (4 spaces).

File: include/robot_interfaces/status.hpp (hunk 1 above)
Comment (vincentberenz) [reply include/robot_interfaces/status.hpp]: indeed

File: include/robot_interfaces/status.hpp (hunk 2 above)
Comment (vincentberenz) [reply include/robot_interfaces/status.hpp]: indeed

**Pull request merged**